### ISSUE TITLE
Allow users to view and manage block/mute lists in settings

### ISSUE THREAD

**Issue by wolfteeth** [mastodon/mastodon#21177]
### Pitch

Add a settings page to allow users to manage lists of users and domains they have blocked or muted. I would be inclined to make this a sub-page under the existing "Filters" tab.

I imagine this looking similar to the "Follows and Followers" page, but with options to filter users by whether they are blocked or muted. It should give the user options to unblock/unmute if desired, or add new users to the blocked/muted list.

Domain blocks could be on a separate page, and make it easy to add or remove domain blocks.

### Motivation

It's currently a bit hard to figure out who you have muted or blocked. You have to go to the Import/Export page and download a csv file. If you want to block/mute or unblock/unmute someone, you have to navigate to that user's profile. I'm not sure if it's possible to unblock a domain at all, and to block it, you have to find the profile of someone there.

It would be nice to have a single page where these settings could be centralized and managed.

**Comment by wolfteeth** [comment]
I just realized this feature does exist, but it's hidden in the ellipsis menu in the main web interface, rather than being part of settings. IMO it should be in settings with the Filters as well.

**Comment by brendanjones** [comment]
Agree, but this is a duplicate of https://github.com/mastodon/mastodon/issues/9650 so you can close it.
